README.md

```diff
@@ -110,36 +131,36 @@ Go back to the GitHub App and update the following settings
 4. Save changes and navigate to the Install App tab
 5. Next to your GitHub App, select Install next to your org and select 'All Repositories'
 
- ## Required Inputs
+## Required Inputs
 
  Below are the required inputs required to get started with this module.  Some may be marked with an asterisk which indicates we recommend you pull this from a data source.  Examples of usage can be found at [Terraform Examples](./modules/terraform-examples).
 
  | Name | Description | Type |
 |------|-------------|------|
-| [azure_tenant_id]() | Azure tenant ID | `string` |
-| [azure_subscription_id]() | Azure subscription ID | `string` |
-| [azure_resource_group_name]() | Resource Group that the components and runners will be created within | `string` |
-| [azure_subnet_id]() | Azure subnet ID | `string` |
-| [name_suffix]() | Identifying suffix that will be appended to all components created by this module (default: `null`) | `string` |
-| [github_organization]() | GitHub organization | `string` |
-| [github_app_id]() | GitHub App ID | `string` |
-| [github_client_id]() | GitHub Client ID | `string` |
-| [github_installation_id]() | GitHub App installation ID | `string` |
-| [azure_secrets_key_vault_resource_id]() | Key Vault ID where GitHub secrets are stored | `string` |
-| [*azure_runner_default_password_key_vault_id]() | Key Vault ID for Azure runner default password ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret))| `string` |
-| [*github_client_secret_key_vault_id]() | Keyvault Vault ID for GitHub App client secret ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
-| [*github_webhook_secret_key_vault_id]() | Keyvault Vault ID for GitHub App webhook secret ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
-| [*github_private_key_key_vault_id]() | Keyvault Vault ID for GitHub App private key ([data source](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/key_vault_secret)) | `string` |
-| [*owners]() | The list of owners that will be assigned to all components ([data source](https://registry.terraform.io/providers/hashicorp/azuread/latest/docs/data-sources/users)) | `list(string)` |
+| <a name="azure_tenant_id"></a> [azure\_tenant\_id](#azure\_tenant\_id) | Azure tenant ID | `string` |
```

Comment:
what's the thought behind the anchors? should these links point to the variable file?

Reply:
The anchors simply allow for someone to copy a link to that specific spot in on the page. They aren't meant to link elsewhere!